set_env_variables.sh

```diff
@@ -8,10 +8,19 @@ log() {
   echo -e "${COLOR_CYAN}$1${COLOR_DEFAULT}"
 }
 
-if [ -z ${SCRIPT_DIR+x} ]; then
-  SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# if the script is run with Jenkins, then set the SCRIPT_DIR manually, otherwise use the old way (SET JENKINS TO 1)
+if [[ -z "${JENKINS}" ]]; then
```

Comment:
What's the source of this variable ?

Reply:
it needs to be set in vigilan sniffle before runing this script, something like "export JENKINS=1"

Reply:
I would recommend using 'IS_JENKINS' var name. It describes situation more clear 🙂

Reply:
I suggest following logic:
 

```
  if IS_JENKINS:
      SCRIPT_DIR="smth"
  elif not SCRIPT_DIR:
      SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```